docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -398,6 +398,8 @@ Now we need to add some more precious information to our bear. This helps
 by giving more information about each bear and also helps some functions
 gather information by using these values. Our bear now looks like:
 
+.. include:: ../Language_Naming_Format.rst
```

Comment:
This does not seem the right place to put this section. Put it under the code snippet below.